nipap-cli/nipap

```diff
@@ -74,7 +74,12 @@ if __name__ == '__main__':
         sys.exit(1)
 
     # read configuration
-    cfg = ConfigParser.ConfigParser()
+    cfg_defaults = {
+        'scheme': 'http',
+        'hostname': 'localhost',
```

Comment:
For those trying out nipap in development or vagrant this is handy, but not necessary.  I can remove it if that's the decision and just leave the port and scheme in the defaults since those should rarely be specified.